org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/WStringEditor.java

```diff
@@ -221,8 +221,10 @@ public void onEvent(Event event)
 	        // Validate VFormat with regular expression
 	        if (!Util.isEmpty(vFormat) && vFormat.startsWith("~")) {
 	        	String regex = gridField.getVFormat().substring(1); // remove the initial ~
-	        	if (!newValue.matches(regex))
-	        		throw new WrongValueException(component, Msg.getMsg(Env.getCtx(), "InvalidFormatRegExp", new Object[] {regex}));
+	        	if (!newValue.matches(regex)) {
+	        		String msgregex = Msg.getMsg(Env.getCtx(), regex);
```

Comment:
anyway to avoid logging of warning here if we doesn't translate the regular expression (I think we don't always do) ?

Reply:
alright, since you feel ok with that, lets move on then.